### DISCUSSION TITLE
Need allure plugin for bamboo version 9.3.1

### DISCUSSION THREAD

**Discussion by manishsoni0407** [allure-framework/allure2#2116] [Questions & Support] [answered]
I am integrating allure with BDD framework, We are using bamboo as CI/CD tool. Moreover after current support for bamboo is till 9.2. However we are using bamboo 9.3.1  

Can i get the plugin for above mentioned version.

**Comment by baev** [comment, marked as answer]
<img width="1083" alt="Screenshot 2023-10-05 at 20 23 44" src="https://github.com/allure-framework/allure2/assets/2149631/43f72d5f-51dd-4455-a4ea-a42881335ef1">

Version 9.3.1 is supported by the plugin
